etc/config/cuda.amazon.properties

```
@@ -442,8 +442,16 @@ libs.cueigen.versions.337.version=3.3.7
 libs.cueigen.versions.337.path=/opt/compiler-explorer/libs/eigen/v3.3.7
 libs.cueigen.versions.334.version=3.3.4
 libs.cueigen.versions.334.path=/opt/compiler-explorer/libs/eigen/v3.3.4
-libs.thrustcub.name=Thrust+CUB
-libs.thrustcub.description=CUDA collective and parallel algorithms
+libs.cccl.name=CUDA C++ Core libraries
+libs.cccl.description=Fundamental CUDA C++ libraries: Thrust, CUB, libcudacxx
+libs.cccl.versions=trunk:2002000
+libs.cccl.url=http://www.github.com/NVIDIA/cccl
+libs.cccl.versions.trunk.version=trunk
+libs.cccl.versions.trunk.path=/opt/compiler-explorer/libs/cccl/trunk/thrust:/opt/compiler-explorer/libs/cccl/trunk/cub:/opt/compiler-explorer/libs/cccl/trunk/libcudacxx/include
+libs.cccl.versions.2002000.version=2.2.0
+libs.cccl.versions.2002000.path=/opt/compiler-explorer/libs/cccl/2.2.0/thrust:/opt/compiler-explorer/libs/cccl/2.2.0/cub:/opt/compiler-explorer/libs/cccl/2.2.0/libcudacxx/include
+libs.thrustcub.name=Thrust+CUB (Legacy)
+libs.thrustcub.description= Use CUDA C++ Core Libraries (CCCL) for newer versions
```

Probably an extra space?
```suggestion
libs.thrustcub.description=Use CUDA C++ Core Libraries (CCCL) for newer versions
```